### PR TITLE
park-api 0.4.3

### PR DESCRIPTION
**Pull request by the-infinity**
New park-api release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
-## [Unreleased]
-
-### Fixes
```
Comment (derhuerst) [inline CHANGELOG.md:9]: Why did you remove this line?

Comment (the-infinity) [reply CHANGELOG.md:9]: Ups. Fixed.

**Review by ThorstenFroehlinghaus — APPROVED** [review]
LGTM

**Review by derhuerst — APPROVED** [review]
